tests/test_translation.h

```diff
@@ -35,6 +35,8 @@
 #include "core/string/translation.h"
 #include "core/string/translation_po.h"
 
+#include "editor/import/resource_importer_csv_translation.h"
```

Comment:
This and the added test should be enclosed in `#ifdef TOOLS_ENABLED`. I think currently we only build tests with the editor so it's always true, but in the future we might want to make it possible to build tests for the export templates too.

Reply:
Added definition check.